Python/PRP/Camera/pyCameraBrain1.cpp

```diff
@@ -0,0 +1,369 @@
+	{
+		return PyFloat_FromDouble(self->fThis->getPanSpeed());
+	}
+
+	static PyObject* pyCameraBrain1_getZoomRate(pyCameraBrain1* self, void*)
+	{
+		return PyFloat_FromDouble(self->fThis->getZoomRate());
+	}
+
+	static PyObject* pyCameraBrain1_getZoomMin(pyCameraBrain1* self, void*)
+	{
+		return PyFloat_FromDouble(self->fThis->getZoomMin());
+	}
+
+	static PyObject* pyCameraBrain1_getZoomMax(pyCameraBrain1* self, void*)
+	{
+		return PyFloat_FromDouble(self->fThis->getZoomMax());
+	}
+
+	static PyObject* pyCameraBrain1_getFlags(pyCameraBrain1* self, void*)
+	{
+		return pyBitVector_FromBitVector(self->fThis->getFlags());
+	}
+
+	static PyObject* pyCameraBrain1_getEoAFlags(pyCameraBrain1* self, void*)
+	{
+		return pyBitVector_FromBitVector(self->fThis->getEoAFlags());
+	}
+
+	static int pyCameraBrain1_setVelocity(pyCameraBrain1* self, PyObject* value, void*)
+	{
+		if (value == NULL || !PyFloat_Check(value)) {
+				PyErr_SetString(PyExc_TypeError, "camera velocity should be a float");
+				return -1;
+		}
+		self->fThis->setVelocity(PyFloat_AsDouble(value));
+		return 0;
+	}
+	
+	static int pyCameraBrain1_setAcceleration(pyCameraBrain1* self, PyObject* value, void*)
+	{
+		if (value == NULL || !PyFloat_Check(value)) {
+			PyErr_SetString(PyExc_TypeError, "camera acceleration should be a float");
+			return -1;
+		}
+		self->fThis->setAcceleration(PyFloat_AsDouble(value));
+		return 0;
+	}
+
+	static int pyCameraBrain1_setDeceleration(pyCameraBrain1* self, PyObject* value, void*)
+	{
+		if (value == NULL || !PyFloat_Check(value)) {
+			PyErr_SetString(PyExc_TypeError, "camera deceleration should be a float");
+			return -1;
+		}
+		self->fThis->setDeceleration(PyFloat_AsDouble(value));
+		return 0;
+	}
+
+	static int pyCameraBrain1_setPOAVelocity(pyCameraBrain1* self, PyObject* value, void*)
+	{
+		if (value == NULL || !PyFloat_Check(value)) {
+			PyErr_SetString(PyExc_TypeError, "camera POAvelocity should be a float");
+			return -1;
+		}
+		self->fThis->setVelocity(PyFloat_AsDouble(value));
+		return 0;
+	}
+
+	static int pyCameraBrain1_setPOAAcceleration(pyCameraBrain1* self, PyObject* value, void*)
+	{
+		if (value == NULL || !PyFloat_Check(value)) {
+			PyErr_SetString(PyExc_TypeError, "camera POAacceleration should be a float");
+			return -1;
+		}
+		self->fThis->setAcceleration(PyFloat_AsDouble(value));
+		return 0;
+	}
+
+	static int pyCameraBrain1_setPOADeceleration(pyCameraBrain1* self, PyObject* value, void*)
+	{
+		if (value == NULL || !PyFloat_Check(value)) {
+			PyErr_SetString(PyExc_TypeError, "camera POAdeceleration should be a float");
+			return -1;
+		}
+		self->fThis->setDeceleration(PyFloat_AsDouble(value));
+		return 0;
+	}
+
+	static int pyCameraBrain1_setPOAOffset(pyCameraBrain1* self, PyObject* value, void*)
+	{
+		if (value == NULL || !pyVector3_Check(value)) {
+			PyErr_SetString(PyExc_TypeError, "camera POAOffset should be a vector3");
+			return -1;
+		}
+		self->fThis->setPOAOffset(*(pyVector3_AsVector3(value)));
```

Comment:
This is essentially performing the `pyVector3_Check` twice (which is why I usually access it directly everywhere else:  `((pyVector3*)value)->fThis`).  If you want to add a wrapper (this is a good thing), you may want to remove the duplicate check -- that is, store the vector pointer off, check it for `NULL`, and print the error in the failed case (or set it in the successful case).